contrib/cli/requirements.txt

```diff
@@ -2,7 +2,7 @@ black[jupyter]==23.3.0 # Linting / code style
 Click>=7.1.2         # CLI tooling
 cookiecutter==2.1.1  # Project templating
 mypy==1.5.1          # Type checker
-pydantic>=1.0,<2.0    # Needed for mypy plugin
+pydantic>=1.0        # Needed for mypy plugin
```

Comment:
Should make me 1.10.8?

Reply:
Actually, I don't understand why we have this entry in cli/requirements at all.
It being in our core requirements should be enough.